apps/ledger-live-mobile/src/context/LedgerStore.tsx

```diff
@@ -121,6 +117,6 @@ export default class LedgerStoreProvider extends Component<
   render() {
     const { children } = this.props;
     const { ready, initialCountervalues } = this.state;
-    return <Provider store={store}>{children(ready, store, initialCountervalues)}</Provider>;
```

Comment:
`store` can be obtained with `useStore` so it's longer needed in the component using this render function (cf. `src/index.tsx`)